### PR TITLE
Dfpl 682 Child recovery order (C18)

### PR DESCRIPTION
**Pull request by chak-shing-lo-justice**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-682

### Change description ###
Dfpl 682 Child recovery order (C18)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/Orders.java [hunk 1]
```diff
@@ -55,7 +60,8 @@ public boolean orderContainsEPO() {
     public boolean isC1Order() {
         return isNotEmpty(orderType) && (this.getOrderType().contains(CHILD_ASSESSMENT_ORDER)
                 || isSecureAccommodationOrder()
-                || isRefuseContactWithChildApplication());
+                || isRefuseContactWithChildApplication()
+                || this.getOrderType().contains(CHILD_RECOVERY_ORDER));
```
Comment (qasimnawaz-moj) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/Orders.java:64]: Should we use the isChildRecoveryOrder() function here instead

Comment (chak-shing-lo-justice) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/Orders.java]: Yes, you are right.

**Review by qasimnawaz-moj — DISMISSED** [review]
LGTM

**Review by qasimnawaz-moj — APPROVED** [review]
LGTM

**Pull request merged**